src/navigation/SwipeNavigator.tsx

```diff
@@ -365,27 +371,56 @@ const TabBar = ({ descriptors, jumpTo, navigation, state }: TabBarProps) => {
   );
 };
 
+function LazyPlaceholder({ route }: { route: RouteProp<ParamListBase, string>['name'] }) {
+  const { isDarkMode } = useColorMode();
+  const { colors } = useTheme();
+
+  const backgroundColor =
+    route === Routes.DAPP_BROWSER_SCREEN || route === Routes.POINTS_SCREEN
+      ? isDarkMode
+        ? BROWSER_BACKGROUND_COLOR_DARK
+        : BROWSER_BACKGROUND_COLOR_LIGHT
+      : colors.white;
+
+  return (
+    <View
+      style={{ backgroundColor, bottom: 0, height: DEVICE_HEIGHT, left: 0, position: 'absolute', right: 0, top: 0, width: DEVICE_WIDTH }}
+    />
+  );
+}
+
+function getTabBar({ descriptors, jumpTo, navigation, state }: MaterialTopTabBarProps) {
+  return <TabBar descriptors={descriptors} jumpTo={jumpTo} navigation={navigation} state={state} />;
+}
```

Comment:
worth memoing these at all?

Reply:
It is but requires quite a few changes to be effective, to shield the tab bar from the updates in `state and `descriptors`.

I've got that work done but don't want to add more to this PR. Will follow up with another one.